tests/rptest/services/redpanda.py

```diff
@@ -1491,6 +1491,7 @@ def set_cluster_config(self,
                            values: dict,
                            expect_restart: bool = False,
                            admin_client: Optional[Admin] = None,
+                           skip_readiness_check=False,
```

Comment:
`skip_readiness_check` isn't the right name. `is_ready` is an internal thing which doesn't make sense outside of the function, it should be  `wait_until_applied=True` or something